platforms/core-runtime/logging/src/integTest/groovy/org/gradle/internal/logging/console/taskgrouping/AbstractConsoleBuildResultFunctionalTest.groovy

```diff
@@ -89,7 +89,8 @@ BUILD SUCCESSFUL in [ \\dms]+
         """
 
         when:
-        succeeds('success')
+        args "--no-problems-report"
```

Comment:
❌ Also, we should have a follow-up issue for maintenance day to get rid of the now-added `--no-problems-report` flags.